frame/fast-unstake/src/tests.rs

```diff
@@ -165,14 +191,14 @@ mod on_idle {
 
 			// set up Queue item
 			assert_ok!(FastUnstake::register_fast_unstake(RuntimeOrigin::signed(2)));
-			assert_eq!(Queue::<T>::get(1), Some(()));
+			assert_eq!(Queue::<T>::get(1), Some(28_000));
```

Comment:
```suggestion
			assert_eq!(Queue::<T>::get(1), Some(Deposit::get()));
```

None of this should be hardcoded.